source/auth/tests/aws_services_lib/README.md

```diff
@@ -0,0 +1,6 @@
+=================
+Mock AWS Services
+=================
+
+This folder contains files from [mongo-enterprise-modules](https://github.com/10gen/mongo-enterprise-modules/tree/master/jstests/external_auth/lib). 
+The files simulate AWS services, and implementers should use them for local testing.
```

Comment:
It looks like `mongo-enterprise-modules` is a private repo, is it OK/necessary to put those files into this public repo? To me it would seem sufficient to provide the above link and some simple instructions about using the `ec2_metadata_http_server.py` script for any drivers engineers who need it.

Reply:
That makes sense. Done